lib/build-cron.js

```diff
@@ -22,8 +23,11 @@ module.exports = function buildCronJobs(cronConfigsDir, collectionName, logClean
 
   if (!cronConfigsDir || !collectionName) {
     const err = new VError('Error: \'cronConfigsDir\' and \'collectionName\' are required params.');
-    raygun.send(err);
-    throw err;
+    return Promise.resolve(raygun.send(err))
```

Comment:
So, that's where I got really bogged down in all this.  So if you look in `build-cron.js`, all the places where we use `.asCallback()` are within the callback function of `cmaster.loadJobs()`.   `raygun.send()` returning a promise was an issue, and I was once again having problem getting the darn error to propagate after being thrown.  The only way around it that I found was to attach the `throw err` as a callback to `raygun.send()` using the `.asCallback()` function of Bluebird